website/docs/r/pi_image.html.markdown

```diff
@@ -79,7 +79,7 @@ Review the argument references that you can specify for your resource.
 - `pi_image_secret_key` - (Optional, String, Sensitive) Cloud Object Storage secret key; required for buckets with private access.
   - `pi_image_secret_key` is required with `pi_image_access_key`
 - `pi_image_storage_pool` - (Optional, String) Storage pool where the image will be loaded, if provided then `pi_image_storage_type` and `pi_affinity_policy` will be ignored.
-- `pi_image_storage_type` - (Optional, String) Type of storage. Will be ignored if `pi_image_storage_pool` or `pi_affinity_policy` is provided. If only using `pi_image_storage_type` for storage selection then the storage pool with the most available space will be selected.
+- `pi_image_storage_type` - (Optional, String) Type of storage. Required if `pi_image_storage_pool` or `pi_affinity_policy` is provided when importing an image from cloud storage.
```

Comment:
I do not understand this change.
As per the description in swagger referred from https://github.com/IBM-Cloud/power-go-client/blob/master/power/models/create_cos_image_import_job.go#L59 this is not changing.

Reply:
@yussufsh This is what the message would be after we GA. It is still the same in SB, I will keep this as it is and would update if it is different than the final message.